### DISCUSSION TITLE
Dev Container Feature at `jpawlowski/devcontainer-features/pnp.powershell`

### DISCUSSION THREAD

**Discussion by jpawlowski** [pnp/powershell#5184] [General]
Hello all,

I wanted to let you know that I took some time to create a so called 'feature' for development containers for PnP.Powershell, which can be found here:

https://github.com/jpawlowski/devcontainer-features/tree/main/src/pnp.powershell

It helps installing the PowerShell module inside [Development Containers](https://containers.dev/) together with the existing VSCode extension.

Let me know if you need any adjustments to the package.

Cheers,
—Julian

**Comment by gautamdsheth** [comment]
Hey @jpawlowski , thank you for taking the time and working on this devcontainer. I looked at it and seems to quite good. Not sure what we can do to enhance it. So, am moving this to discussion forum here.
